index.html

```diff
@@ -65,10 +65,30 @@ <h2>preferCurrentTab</h2>
         </dt>
         <dd>
           <p>
-            When {{MediaDevices/getDisplayMedia()}} is called, if <code>preferCurrentTab</code> is
-            <code>true</code>, the user agent SHOULD present the current tab as the most prominent
-            option. Otherwise, no such requirement is made.
+            When {{MediaDevices/getDisplayMedia()}} is called, the user agent MUST run the following
+            steps:
           </p>
+          <ol>
+            <li>
+              <p>Let <var>constraints</var> be the method's first argument.</p>
+            </li>
+            <li>
+              If <code>constraints.preferCurrentTab</code> is <code>false</code>, abort these steps
+              and continue instead with the usual set of steps associated
+              {{MediaDevices/getDisplayMedia()}}.
+            </li>
+            <li>
+              If <code>constraints.</code>{{DisplayMediaStreamOptions/selfBrowserSurface}} is
+              {{SelfCapturePreferenceEnum/"exclude"}}, return a promise [=reject|rejected=] with a
+              newly [=exception/created=] {{TypeError}}.
+            </li>
+            <li>
+              All other steps associated with {{MediaDevices/getDisplayMedia()}} should now be
```

Comment:
The "all other steps" part is not clear. Can you call the relevant algorithm here?

Reply:
Done.